Don't download Google Test. Allow an already installed Google Test to be used.
https://github.com/Boolector/boolector/commit/91533caf29a2c5b10b4912fd352e7af82c787598
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -440,13 +440,6 @@ configure_file(
 #-----------------------------------------------------------------------------#
 # Regression tests
 
-# Get and configure google test
-include(cmake/googletest.cmake)
-fetch_googletest(
-    ${PROJECT_SOURCE_DIR}/cmake
-    ${PROJECT_BINARY_DIR}/googletest
-    )
-
 enable_testing()
 
 #-----------------------------------------------------------------------------#
--- /dev/null
+++ cmake/FindGoogleTest.cmake
@@ -0,0 +1,1 @@
+find_package(GTest 1.10.0)
--- test/CMakeLists.txt.orig
+++ test/CMakeLists.txt
@@ -5,6 +5,9 @@
 # This file is part of Boolector.
 # See COPYING for more information on using this software.
 #
+
+find_package(GoogleTest REQUIRED)
+
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/tests)
 
 set(test_names
@@ -47,7 +50,7 @@ foreach(test ${test_names})
   add_executable (test${test} test_${test}.cpp)
   target_include_directories(test${test} PRIVATE ${PROJECT_SOURCE_DIR}/test/new_test)
   target_link_libraries(test${test} boolector m)
-  target_link_libraries(test${test} gtest_main)
+  target_link_libraries(test${test} GTest::gtest_main)
   set_target_properties(test${test} PROPERTIES OUTPUT_NAME test${test})
   add_test(${test} ${CMAKE_BINARY_DIR}/bin/tests/test${test})
 endforeach()
